### ISSUE TITLE
PNG and RAW capture

### ISSUE THREAD

**Issue by M-Viccars** [waveform80/compoundpi#55]
It would be great to have PNG and RAW capture options, for projects that need detail over speed.

**Comment by jojo200** [comment]
Yes, png would be great. I tried to hack it myself but I failed.
